modules/ROOT/pages/composefs.adoc

```diff
@@ -0,0 +1,46 @@
+= ComposeFS
+
+Fedora CoreOS introduced ComposeFS enabled by default starting in Fedora 41. ComposeFS is an overlay filesystem where the data comes from the usual ostree deployement, and
+metadata are in the composeFS file. The result is a truely read-only root (`/`) filesystem, increasing the system integrity and robustness,
```

Comment:
Upstream mostly uses `composefs` (https://github.com/search?q=repo%3Acontainers%2Fcomposefs%20composefs&type=code) only uses `Composefs` at the beginning of sentences, so let's use `composefs` everywhere.